app/ui-react/syndesis/src/modules/data/shared/VirtualizationUtils.ts

```diff
@@ -417,10 +417,13 @@ export function getPreviewSql(viewDefinition: ViewDefinition): string {
 function getPreviewTableName(sourcePath: string): string {
   // Assemble the name, utilizing the schema model suffix
   return (
+    '"' +
     getConnectionName(sourcePath).toLowerCase() +
     SCHEMA_MODEL_SUFFIX +
-    '.' +
-    getNodeName(sourcePath)
+    '".' +
+    '"' +
+    getNodeName(sourcePath) +
+    '"'
```

Comment:
You could do:

```
`"${getConnectionNAme(sourcePath).toLowerCase()}${SCHEMA_MODEL_SUFFIX}.${getNodeName(sourcePath)}"."${getNodeName(sourcePath)"`
```
Using a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) if you'd rather avoid all the `+` etc.  Depends on what you think is easier to read, I've been finding though that the template literals make me more comfortable to use than building complex strings like this.

Reply:
Yeah, agree it would be easier to read.  I can look at changing in next commit. Thx!